fix-point/fix-point.cabal

```diff
@@ -26,14 +20,18 @@ source-repository head
 library
   exposed-modules:
       Rename
+      CopyFiles
   other-modules:
       Paths_fix_point
   hs-source-dirs:
       src
   build-depends:
       base >=4.7 && <5
     , text
-  ghc-options: 
+    , directory
+    , filepath
+    , filesystem-trees 
```

Comment:
Since you're not using this package, it's better to remove it from `build-depends`